.travis.yml

```diff
@@ -65,16 +65,16 @@ cache:
   - packages/variable-resolver/node_modules
   - packages/workspace/node_modules
 # end_cache_directories
-  - packages/java-debug/download
-  - packages/debug-nodejs/download
+    - packages/java-debug/download
```

Comment:
```
ERROR:

while parsing a block mapping
  in "<unicode string>", line 7, column 3:
      yarn: true
      ^
expected <block end>, but found '<block sequence start>'
  in "<unicode string>", line 68, column 5:
        - packages/java-debug/download
        ^
```

This specific indentation breaks the way the yaml is parsed. You should edit the `scripts/prepare-travis` script to autogen the correct indentation level?

Reply:
Oh, thank you @marechal-p

Reply:
Or we change yaml settings to use 2 space identation like for json files and avoid whitespace changes @marechal-p sounds good?

Reply:
ah, it is already 2 spaces, so will fix prepare-travis